infrastructure/stacks/cloudwatch-dashboard/main.tf

```diff
@@ -0,0 +1,141 @@
+resource "aws_cloudwatch_dashboard" "cloudwatch_dashboard" {
+
+  dashboard_name = var.cloudwatch_performance_dashboard_name
+  dashboard_body = <<EOF
+{
+    "widgets": [
+        {
+            "height": 6,
+            "width": 23,
+            "y": 0,
+            "x": 0,
+            "type": "metric",
+            "properties": {
+                "view": "timeSeries",
```

Comment:
cant see the metric mentioned here just the dimension, shouldnt it say ProcessingLatency somewhere

Reply:
This has been updated which you can see here - https://eu-west-2.console.aws.amazon.com/cloudwatch/home?region=eu-west-2#dashboards:name=uec-dos-int-perf-performance

As ProcessingLatency metric doesn't exist anymore